config/db.connect.js

```diff
@@ -0,0 +1,21 @@
+// package imports
+const mongoose = require('mongoose');
+const to = require('await-to-js').default;
+
+// Logger import
+const logger = require('./logger/logger');
+
+// Constant imports
+const constants = require('./constants');
+
+const dbConnect = async () => {
+  const [err, success] = await to(mongoose.connect(process.env.MONGODB_URL, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
```

Comment:
```suggestion
    useUnifiedTopology: true,
```
Please explain why are you using this?

Reply:
it is a type of mechanism that mongoose uses to establish connection with mongodb servers. In this mechanism/engine, the pool of connections is created that can be shared across all mongodb servers. it improves the mongodb performance.

Reply:
Ack

Reply:
Done